book/website/reproducible-research/rdm/rdm-repository.md

```diff
@@ -0,0 +1,159 @@
+- An introduction to the Open Science Framework
+
+(rr-rdm-repository-FAIR)=
+## Repositories and FAIR
+
+Selecting an appropriate repository for your research outputs has many benefits:
+- It helps make your Research Objects more FAIR ({ref}`Findable, Accessible, Interoperable and Reusable<rr-rdm-fair>`). This is achieved through:
+    - Repositories assign a Persistent Identifier to your Research Objects, which makes them findable and citable (see {ref}`Citing Research Objects<cm-citable-cite>`.
+The most commonly used persistent identifiers for research objects is the Digital Object Identifier, usually abbreviated to DOI.
+    - Repositories use metadata standards in describing your Research Object, which ensures that other people can find it using search engines.
+    - Repositories add a licence to the Research Objects.
+A {ref}`license <rr-licensing>` describes to potential reusers of your work what they are allowed to do with it. 
+    - Repositories provide documentation for Research Objects.
+This can be in the form of READMEs and/or wikis that provide a description of your project and why it might be relevant to people.
+    - Encouraging widely-used file formats.
+Many repositories have restrictions on the file formats used to ensure the sustainability of Research Objects.
+Some file formats (especially proprietary ones with a limited user base) can become deprecated.
+- It allows to determine the levels of access to Research Objects.
+As covered in {ref}`Barriers to Data Sharing<rr-open-data-barriers>`, there are good reasons to not to make all Research Objects completely open.
+However, it's still worthwhile to at least open the metadata and provide an option for people to obtain access to the actual Research Objects if they have certain credentials or if they have been given explicit access.
+That way, your work will still be FAIR (because the metadata are findable and there is an access procedure in place), as well as and secure (because you can control who has access).
+    - Restricting access and storing data on European servers can help to manage sensitive data {ref}`manage sensitive data<pd-sdpm>`
+
+## Why not the supplemental materials?
+
+Supplemental materials are not following the FAIR principles - as there is no seperate DOI assigned to the supplemental materials which makes it difficult to retrieve these materials. 
+Next to supplemental materials not being aligned with the FAIR principles, there are other reasons why a data repository is a better solution: 
+
+- Data control: supplementary materials cannot be updated, unlike materials available at data repositories.
+- Interoperability: If publishers only allow text and PDF formats it hampers data sharing and it will be difficult to reuse the data.
+- Availability: supplementary materials are difficult to access if the article is behind the paywall, and links to supplementary materials can break (since they do not have their own persistent identifier).
```

Comment:
```suggestion
- Availability: Supplementary materials are difficult to access if the article is behind the paywall, and links to supplementary materials can break (since they do not have their own persistent identifier).
```